### PR TITLE
Swift 6 consideration: make SFSymbol Sendable

### PR DESCRIPTION
**Pull request by Dean151**
## Motivation
Swift 6 is coming (one day?) and Strict Concurrency Checking is the new safe default to build new projects.
Since latest Xcode beta (15.3 beta 2), Strict Concurrency warnings are becoming more powerful … and trigger warnings on SFSymbols and related not being sendable.

## An easy fix
Therefore, the fix is easy. A class is sendable when it's immutable and not overridable, or when it manage its internal state safely. In our case, we don't have the "not overridable", but it's manageable! The class only store immutable data (rawValue) and it's not overridable outside of the module, because it's not open. Therefore, we can safely assume it's Sendable by default, and add `@unchecked Sendable`.
Unchecked is a little bit misnamed, maybe? It's not that we haven't checked as a developer. It means the compiler hadn't checked it. 
And like we say. With great power comes great responsibility.

### CODE REVIEW

**Comment by StevenSorial** [comment]
Thanks for the PR. can you please rebase before I can take a look?

**Comment by Dean151** [comment]
Hi @StevenSorial !
The rebase is complete !
Have a nice review :)

File: Sources/SFSafeSymbols/Symbols/SymbolLocalizations.swift [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // Don't touch this manually, this code is generated by the SymbolsGenerator helper tool
 
-public enum Localization: String, Equatable {
+public enum Localization: String, Equatable, Sendable {
```
Comment (StevenSorial) [inline Sources/SFSafeSymbols/Symbols/SymbolLocalizations.swift]: Correct me if I'm wrong, but I don't think simple enums need an explicit conformance.
if that's the case then we can remove it

**Comment by StevenSorial** [comment]
Thanks. I left one comment.
Could you also please add a changelog entry with that change?

**Comment by Dean151** [comment]
@StevenSorial I can't see your comment yet, is it still "pending for sending" in the review feature of Github?

**Review by StevenSorial — CHANGES_REQUESTED** [review]

File: Sources/SFSafeSymbols/Symbols/SymbolLocalizations.swift (hunk 1 above)
Comment (Dean151) [reply Sources/SFSafeSymbols/Symbols/SymbolLocalizations.swift]: I agree. Since String is Sendable, so is the enum, implicitly. Removing it right away!

Comment (Dean151) [reply Sources/SFSafeSymbols/Symbols/SymbolLocalizations.swift]: And it's done :)

**Comment by StevenSorial** [comment]
Thanks for the changes. Feel free to credit yourself in the changelog using the same format as the rest.
I will merge the PR tomorrow.

**Pull request merged**